metarecord/management/commands/import_attributes.py

```diff
@@ -16,7 +16,8 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         filename = options['filename']
         try:
-            tos_importer = TOSImporter(filename)
+            tos_importer = TOSImporter(options)
+            tos_importer.open(filename)
```

Comment:
If the command didn't work, doesn't it mean that it wasn't used anymore? Should we just remove it then?

Or do you know the story behind this? How was this command even supposed to be used and in which situation?

Reply:
Don't know too much about this command other that it did not work when I tested it. I can ask about it.

Reply:
Maybe we can remove it later. Let's at least merge the new tests first so that they get to be part of the Git history for future reference at least (if someone is going to write a new importer based on that one).